src/Analysis.cpp

```diff
@@ -118,10 +118,11 @@ void buildCFG(CFG &topCFG, std::vector<std::string> branchOrder,
     for (auto succ : succs) {
       std::string succName = succ->getParent()->getName().str();
 
-      if (succName == branchName)
+      if (succName == branchName) {
         continue;
+      }
 
-      if (cfgMap.count(succName) > 0) {
+      if (cfgMap.count(succName)) {
```

Comment:
I prefer the explicit "> 0" as a programmer who works primarily in not-C/C++, but if this is idiomatic the change is ok.

Reply:
A brief internet search suggests that this is idiomatic in C++, so this change can stand.